src/boot/boot.c

```diff
@@ -2970,6 +3005,7 @@ static EFI_STATUS run(EFI_HANDLE image) {
         _cleanup_free_ char16_t *loaded_image_path = NULL;
         (void) device_path_to_str(loaded_image->FilePath, &loaded_image_path);
         config_load_all_entries(&config, loaded_image, loaded_image_path, root_dir);
+        sysfail_process(&config);
```

Comment:
cast to void if you ignore the return value knowingly. alternatively make the func void

Reply:
fixed, thanks!